specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/sshPublicKeyExamples/SshPublicKey_GenerateKeyPair_EncryptionWithRSA.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-09-01",
+    "sshPublicKeyName": "mySshPublicKeyName",
+    "parameters": {
+      "encryptionType": "RSA"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "privateKey": "{ed25519 private key}",
```

Comment:
>      "privateKey": "{ed25519 private key}", [](http://example.com/codeflow?start=2&length=44)

You need to declare this as x-ms-secret in the swagger, where is that?

Reply:
same as above